services/auth/test/azure_test.js

```diff
@@ -1,307 +0,0 @@
-const assert = require('assert');
-const helper = require('./helper');
-const _ = require('lodash');
-const azure = require('fast-azure-storage');
-const taskcluster = require('taskcluster-client');
-const testing = require('taskcluster-lib-testing');
-
-helper.secrets.mockSuite(testing.suiteName(), ['db', 'azure', 'gcp'], function(mock, skipping) {
-  if (mock) {
-    return; // We only test this with real creds
-  }
-  helper.withCfg(mock, skipping);
-  helper.withDb(mock, skipping);
-  helper.withPulse(mock, skipping);
-  helper.withEntities(mock, skipping);
-  helper.withServers(mock, skipping);
-  helper.resetTables(mock, skipping);
-
-  let testaccount;
-  suiteSetup('get azure test account name', async function() {
-    const cfg = await helper.load('cfg');
-    testaccount = _.keys(cfg.app.azureAccounts)[0];
-  });
-
-  test('azureAccounts', function() {
-    return helper.apiClient.azureAccounts().then(function(result) {
-      assert.deepEqual(result.accounts, _.keys(helper.cfg.app.azureAccounts));
-    });
-  });
-
-  test('azureTables', async function() {
-    // First make sure the table exists
-    await helper.apiClient.azureTableSAS(
-      testaccount,
-      'TestTable',
-      'read-write',
-    );
-    let extra = {};
-    do {
-      const result = await helper.apiClient.azureTables(testaccount, extra);
```

Comment:
This method and the others tested here still exist, so I think we need to keep this file :)